x-pack/docs/en/ilm/update-lifecycle-policy.asciidoc

```diff
@@ -1,5 +1,432 @@
+// TESTRESPONSE[s/"action_time": 1538475653317/"action_time": $body.indices.my_index.action_time/]
+// TESTRESPONSE[s/"step_time": 1538475653317/"step_time": $body.indices.my_index.step_time/]
+// TESTRESPONSE[s/"modified_date_in_millis": 1538475653317/"modified_date_in_millis": $body.indices.my_index.phase_execution.modified_date_in_millis/]
+
+Updating `my_executing_policy` to have no rollover action and, instead, go directly into a newly introduced `warm` phase.
+
+[source,js]
+------------------------
+PUT _ilm/my_executing_policy
+{
+  "policy": {
+    "phases": {
+      "warm": {
+        "minimum_age": "1d",
+        "actions": {
+          "forcemerge": {
+            "max_num_segments": 1
+          }
+        }
+      },
+      "delete": {
+        "minimum_age": "10d",
+        "actions": {
+          "delete": {}
+        }
+      }
+    }
+  }
+}
+------------------------
+// CONSOLE
+// TEST[continued]
+
+Now, version 2 of this policy has no `hot` phase, but if we run the Explain API again, we will see that nothing has changed.
+
+////
+[source,js]
+--------------------------------------------------
+GET my_index/_ilm/explain
+--------------------------------------------------
+// CONSOLE
+// TEST[continued]
+////
+
+[source,js]
+--------------------------------------------------
+{
+  "indices": {
+    "my_index": {
+      "index": "my_index",
+      "managed": true,
+      "policy": "my_executing_policy",
+      "skip": false,
+      "lifecycle_date": 1538475653281,
+      "phase": "hot",
+      "phase_time": 1538475653317,
+      "action": "rollover",
+      "action_time": 1538475653317,
+      "step": "attempt_rollover",
+      "step_time": 1538475653317,
+      "phase_execution": {
+        "policy": "my_executing_policy",
+        "modified_date_in_millis": 1538475653317,
+        "version": 1,
+        "phase_definition": {
+          "minimum_age": "0ms",
+          "actions": {
+            "rollover": {
+              "max_docs": 1
+            }
+          }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TESTRESPONSE[s/"lifecycle_date": 1538475653281/"lifecycle_date": $body.indices.my_index.lifecycle_date/]
+// TESTRESPONSE[s/"phase_time": 1538475653317/"phase_time": $body.indices.my_index.phase_time/]
+// TESTRESPONSE[s/"action_time": 1538475653317/"action_time": $body.indices.my_index.action_time/]
+// TESTRESPONSE[s/"step_time": 1538475653317/"step_time": $body.indices.my_index.step_time/]
+// TESTRESPONSE[s/"modified_date_in_millis": 1538475653317/"modified_date_in_millis": $body.indices.my_index.phase_execution.modified_date_in_millis/]
+
+After indexing one document into `my_index` so that rollover succeeds and moves onto the next phase, we will notice something new.
+
+////
+[source,js]
+--------------------------------------------------
+PUT my_index/_doc/1
+{
+  "foo": "bar"
+}
+
+GET my_index/_ilm/explain
```

Comment:
I have removed the `WIP` label because things are way more complete, but still not 100%.

Do you have any recommendations as to how to force the policy to move on between phases here in a timely manner? I would rather not use examples that are misleading and explicitly defined such that the policy is stuck in any action. I was hoping that the AsyncWaitStep of rollover would kick-in in time to move to the next step.

Reply:
There isn't a way to do this in the docs tests right now. I think we will have to add `TESTRESPONSE[skip:some reason]` here for now as I don't see a way to get these to work

Reply:
will do that then